doc/cupertino-controls-styles.md

```diff
@@ -0,0 +1,21 @@
+# Cupertino Controls Styles
```

Comment:
ignore other comment in this file... and just paste this in:
```md
`Button`|CupertinoButtonStyle<br/>CupertinoContainedButtonStyle<br/>CupertinoDatePickerFlyoutButtonStyle
`CalendarDatePicker`|CupertinoCalendarDatePickerStyle
`CalendarView`|CupertinoCalendarViewStyle
`CheckBox`|CupertinoCheckBoxStyle<br/>CupertinoSecondaryCheckBoxStyle
`ComboBox`|CupertinoComboBoxStyle
`ComboBoxItem`|CupertinoComboBoxItemStyle
`DatePicker`|CupertinoDatePickerStyle
`DatePickerFlyoutPresenter`|CupertinoDatePickerFlyoutPresenterStyle
`HyperlinkButton`|CupertinoHyperlinkButtonStyle
`muxc:NumberBox`|CupertinoNumberBoxStyle
`muxc:ProgressBar`|CupertinoProgressBarStyle
`muxc:ProgressRing`|CupertinoProgressRingStyle
`PasswordBox`|CupertinoPasswordBoxStyle
`RadioButton`|CupertinoRadioButtonStyle
`Slider`|CupertinoSliderStyle<br/>CupertinoSecondarySliderStyle
`TextBlock`|CupertinoBaseTextBlockStyle<br/>CupertinoLargeTitle<br/>CupertinoPrimaryTitle<br/>CupertinoSecondaryTitle<br/>CupertinoTertiaryTitle<br/>CupertinoHeadline<br/>CupertinoBody<br/>CupertinoCallout<br/>CupertinoSubhead<br/>CupertinoFootnote<br/>CupertinoPrimaryCaption<br/>CupertinoSecondaryCaption
`TextBox`|CupertinoTextBoxStyle
`Thumb`|CupertinoSliderThumbStyle
`ToggleSwitch`|CupertinoToggleSwitchStyle
```

Reply:
maybe do a sanity check

Reply:
thanks !